components/cards.css

```diff
@@ -0,0 +1,135 @@
+.card {
+  max-width: 18.75rem;
+  background-color: #171717;
+  border-radius: 5px;
+  text-align: center;
+
```

Comment:
remove unnecessary space

Reply:
thank you for pointing this out, this was intentional by me to kind of highlight the position value of this css class